media/css/firefox/developer-quantum-whatsnew.scss

```diff
@@ -0,0 +1,181 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import
+    '../pebbles/includes/lib';
+
+$base-blue: #306ffd;
+
+@mixin section-curve {
+    position: absolute;
+    display: block;
+    left: 0;
+    content: '';
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    width: 100%;
+    visibility: visible;
+    z-index: 10;
+}
+
+//*------------------------------------------------------------------*/
+// Intro
+.section-intro {
+    background-color: #fff;
+    background-position: center bottom;
+    background-repeat: no-repeat;
+    text-align: center;
+
+    @media #{$mq-tablet} {
+        @include background-size(1440px 1034px);
+        background-image: url('/media/img/firefox/products/developer/quantum/whatsnew-hero-background.png');
+        background-position: 0 0;
+        padding-bottom: 40px;
+
+        &:after {
+            @include background-size(3000px 62px);
+            @include section-curve;
+            background-image: url('/media/img/firefox/products/developer/quantum/curve-download-bottom-whatsnew.svg');
+            height: 62px;
+            bottom: -1px;
+        }
+    }
+
+    @media #{$mq-desktop} {
+        @include at2x('/media/img/firefox/products/developer/quantum/whatsnew-hero-background.png', 1780px, 1278px);
+        background-position: 0 -169px;
```

Comment:
Centered here, too, and probably anchored to the bottom to better handle vertical expansion, though that makes the positioning trickier... Maybe just leave it as `center bottom` and let the tops of the blobs be cut off.